libhb/rpu.c

```diff
@@ -284,7 +283,16 @@ static int rpu_work(hb_filter_object_t *filter,
 
             if (pv->mode)
             {
-                const DoviData *rpu_data = dovi_write_unspec62_nalu(rpu_in);
+                const DoviData *rpu_data = NULL;
+
+                if (pv->mode & RPU_MODE_EMIT_UNSPECT_62_NAL)
+                {
+                    rpu_data = dovi_write_unspec62_nalu(rpu_in);
+                }
+                else if (pv->mode & RPU_MODE_EMIT_T35_OBU)
+                {
+                    rpu_data = dovi_write_av1_rpu_metadata_obu_t35_payload(rpu_in);
```

Comment:
For AV1 SVT needs the buffer to contain the country code, so `dovi_write_av1_rpu_metadata_obu_t35_complete` exists for that.
See also: https://github.com/quietvoid/SVT-AV1/commit/869d2f4e73c7d18afc41e62b25a8d45390b91db5#diff-955c54ae7f5deccff4de64165b926a83321a0cbcc181a068d6f4bc76f3eff9e9R471

Reply:
Oh right, fixed.
I tested with FFmpeg git master, and libdav1d parses the written RPUs now.